src/test/kotlin/com/cosmotech/api/security/keycloak/KeycloakJwtAuthenticationConverterTests.kt

```diff
@@ -27,19 +27,20 @@ class KeycloakJwtAuthenticationConverterTests {
   @BeforeTest
   fun beforeEachTest() {
     logger.trace("Begin test")
-    csmPlatformProperties = mockk<CsmPlatformProperties>(relaxed = true)
-    jwt = mockk<Jwt>(relaxed = true)
```

Comment:
I'm not sure about the behaviour of relaxed, is it not to avoid exception when all methods of the mocked class are not all 'mocked'?
Why changing this?

Reply:
Regarding documentation [here](https://mockk.io/#relaxed-mock), `relaxed mockk` defined simple value for all mockk functions. 
I'm not a huge fan of this, cause it allows developers to not take care of what they're mocking.
Plus, `relaxed` properties was misused and overused in all tests (sometimes defined without any reason)
In this specific test, once `relaxed` property removed, I just had to define two behaviors ( `csmPlatformProperties.authorization.principalJwtClaim` and `jwt.getClaimAsString(principalClaimValue)`). It's not a big deal and it allows to have a fine tune in mock behavior in our tests